docs/sources/getting-started/add-preconfigured-dashboard.md

```diff
@@ -1,33 +1,32 @@
 +++
 aliases = ["/docs/grafana/latest/getting-started/getting-started/", "/docs/grafana/latest/guides/getting_started/", "/docs/grafana/latest/guides/gettingstarted/"]
-description = "Guide for getting started with Grafana"
-keywords = ["grafana", "intro", "guide", "started"]
-title = "With Grafana"
+description = "Learn how to get started with Grafana by adding a preconfigured dashboard."
+title = "Add a preconfigured dashboard"
 weight = 200
 +++
 
-# Getting started with Grafana
+# Add a preconfigured dashboard
```

Comment:
I do not understand this title at all, this guide describes how to create new dashboard, add a new panel with a test data source (I SUPER stupid pointless guide). 

This guide describes nothing about importing an existing dashboard (preconfigured). It's a bit pointless to be honest.

Reply:
> I do not understand this title at all, this guide describes how to create new dashboard, add a new panel with a test data source (I SUPER stupid pointless guide).
> 
> This guide describes nothing about importing an existing dashboard (preconfigured). It's a bit pointless to be honest.

Plus 1

Reply:
But it is a good guide as it describes how to first access and login to grafana (so linked to from install pages I see)